src/converters/common.ts

```diff
@@ -6,52 +6,26 @@ export function hasImages(name: string) {
   return name.includes('![](https://');
 }
 
-const months = [
-  'January',
-  'February',
-  'March',
-  'April',
-  'May',
-  'June',
-  'July',
-  'August',
-  'September',
-  'October',
-  'November',
-  'December',
-];
-
-const monthPrefixes = months.map((m) => m.slice(0, 3));
-
-// Convert 'June 1st, 2021' and 'Jun 1st, 2021' to 'MM-DD-YYYY' without dealing with timezones, etc.
-export function dateStringToUSDateUID(str: string) {
-  str = str.replace(/(^\w+\s\d{1,2})(\w{2}),(\s\d+)/, '$1$3');
-  const pieces = str.split(/\s/);
-
-  const monthMatch = months.indexOf(pieces[0]);
-  const monthPrefixMatch = monthPrefixes.indexOf(pieces[0]);
-  let month;
-  if (monthMatch !== -1) {
-    month = monthMatch + 1;
-  } else if (monthPrefixMatch !== -1) {
-    month = monthPrefixMatch + 1;
-  } else {
-    return str;
-  }
-
-  return `${month.toString().padStart(2, '0')}-${pieces[1].toString().padStart(2, '0')}-${pieces[2]
-    .toString()
-    .padStart(4, '0')}`;
+/**
+ * Convert a date to US MM-DD-YYYY format, which is used in Tana for journal pages.
+ * @param date
```

Comment:
Ow. That's very legacy, and should be removed. That insane format should not be profilated.

We use ISO-standard for all date-stuff. So Y-m-d.

Reply:
Great! I've added another commit to have the dates uniformly in the YYYY-MM-DD format.